### DISCUSSION TITLE
How to backup information?

### DISCUSSION THREAD

**Discussion by bsouza-site** [netbox-community/netbox#9452] [General]
Hello everybody.

I didn't find it in the official documentation.

I will need to change the server where netbox is installed, how can I backup the information present in the system?

**Comment by bogdancordos** [comment]
Administration -> Replicating NetBox:
https://docs.netbox.dev/en/stable/administration/replicating-netbox/

You should find all the information need here.

**Reply by bsouza-site** [reply]
> 

Thanks, I will check this information!

**Comment by candlerb** [comment]
For the database, here's a simple script you can run from cron:

```
#!/bin/sh -e
mkdir -p /var/backups/netbox
chmod 750 /var/backups/netbox
su -c 'pg_dump netbox' postgres | gzip -9 >/var/backups/netbox/netbox.$(date +%Y%m%d%H%M).gz
```

**Reply by bsouza-site** [reply]
Thanks, I will run a test.
